### PR TITLE
Fix links and titles in README

### PR DESCRIPTION
**Pull request by leouieda**
Also update the journal citation and DOI link
and include link to PDF at pinga-lab.org.

cc @birocoles @andrelreis

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +15,11 @@ and Pereira, J. M. B.<sup>3</sup>
...
 
 This paper has been published in 
-[**Geochemistry, Geophysics, Geosystems (G<sup>3</sup>)**](https://doi.org/10.1002/2016GC006329).
+[**Geochemistry, Geophysics, Geosystems (G<sup>3</sup>)**](http://agupubs.onlinelibrary.wiley.com/hub/journal/10.1002/(ISSN)1525-2027).
 
-doi: [10.1002/2016GC006329](https://doi.org/10.1002/2016GC006329)
+doi: [10.1002/2016GC006329](http://onlinelibrary.wiley.com/doi/10.1002/2016GC006329/abstract)
 
-A PDF version is available at [pinga-lab.org/papers/paper-magnetization-rock-sample-2016.html](http://www.pinga-lab.org/papers/paper-magnetization-rock-sample-2016.html)
+A PDF version is available at [pinga-lab.org/pdf/paper-magnetization-rock-sample-2016.pdf](http://www.pinga-lab.org/pdf/paper-magnetization-rock-sample-2016.pdf)
```
Comment (leouieda) [inline README.md]: @andrelreis Don't link directly to PDFs. They aren't part of the group page and so people won't be able to see other works by the group. That's the point of the website. When someone visits they can click on your name and see what else you published.

File: README.md [hunk 2]
```diff
@@ -14,11 +15,11 @@ and Pereira, J. M. B.<sup>3</sup>
 de Janeiro, RJ, Brazil.*
 
 This paper has been published in 
-[**Geochemistry, Geophysics, Geosystems (G<sup>3</sup>)**](https://doi.org/10.1002/2016GC006329).
+[**Geochemistry, Geophysics, Geosystems (G<sup>3</sup>)**](http://agupubs.onlinelibrary.wiley.com/hub/journal/10.1002/(ISSN)1525-2027).
 
-doi: [10.1002/2016GC006329](https://doi.org/10.1002/2016GC006329)
+doi: [10.1002/2016GC006329](http://onlinelibrary.wiley.com/doi/10.1002/2016GC006329/abstract)
```
Comment (leouieda) [inline README.md]: @andrelreis also don't use the publisher link if you can use the DOI. The purpose of the DOI link is that it never changes. If the publisher changes their website, your link will be broken. For example, if AGU go away from Wiley.

Comment (andrelreis) [inline README.md]: @leouieda  How can I find the DOI link?

File: README.md (hunk 1 above)
Comment (andrelreis) [inline README.md]: @leouieda Hi Leo! You mean that I have to direct the link to my personal page on pinga-lab?

File: README.md (hunk 2 above)
Comment (leouieda) [inline README.md]: You put `https://doi.org/` and the DOI. Like I had done.

File: README.md (hunk 1 above)
Comment (leouieda) [inline README.md]: No. You link to the publication page.

You changed `http://www.pinga-lab.org/papers/paper-magnetization-rock-sample-2016.html` to `http://www.pinga-lab.org/pdf/paper-magnetization-rock-sample-2016.pdf` for some reason. Why?

Comment (leouieda) [inline README.md]: For example: http://www.pinga-lab.org/papers/paper-magnetization-rock-sample-2016.html

This has a lot more information than the PDF. It has links to your pages, links to the github repository, article level metrics, and citation.

**Comment by andrelreis** [comment]
@leouieda Everything ok now?

**Comment by leouieda** [comment]
@andrelreis should be safe to merge whenever you want

**Pull request merged**